src/Apps/Sell/Routes/PurchaseHistoryRoute.tsx

```diff
@@ -11,15 +20,30 @@ import { useSellFlowContext } from "Apps/Sell/SellFlowContext"
 const FRAGMENT = graphql`
   fragment PurchaseHistoryRoute_submission on ConsignmentSubmission {
     provenance
+    signature
   }
 `
 
 const Schema = Yup.object().shape({
   provenance: Yup.string().trim(),
+  signature: Yup.boolean(),
 })
 
+export const PROVENANCE_LIST = [
+  "Purchased directly from gallery",
+  "Purchased directly from artist",
+  "Purchased at auction",
+  "Gift from the artist",
+  "Other",
+  "I don’t know",
+].map(provenance => ({
+  value: provenance,
+  text: provenance,
+}))
+
 interface FormValues {
-  provenance
+  provenance: string
+  signature: boolean | undefined
```

Comment:
```suggestion
  signature: boolean | null
```

Reply:
Good idea to match the interface with the Metaphysics schema. The only problem could be that the input components might not accept `null` values.